packages/core/content-releases/server/src/services/release.ts

```diff
@@ -619,6 +625,8 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
       },
     });
 
+    strapi.telemetry.send('didPublishContentRelease');
```

Comment:
We already have this event being tracked (as well as for creating a release) in the front with different names "didCreateRelease" and "didPublishRelease"

Adding this with a different name in the backend is on purpose? (To compare vs the front one) Or should we keep only one?

Reply:
@Feranchz Did you test them? I confirm that we don't receive any events named like this on our systems. In our case, they may be implemented, but they don't work.